src/main/java/com/github/fabienrenaud/jjb/support/BenchSupport.java

```diff
@@ -32,6 +33,23 @@
         new Libapi(Library.TAPESTRY, Api.STREAM),
         new Libapi(Library.JSONITER, Api.DATABIND),
         new Libapi(Library.MINIMALJSON, Api.STREAM)
+    ),
+    CLIENTS(
```

Comment:
it's a little difficult to see which libraries are excluded for incompatibilities.
would you mind rewriting this enum to default to all known LibApi implementations and have its constructor take a list of disabled Library + LibApi... this would help on the reading and maintenance.